{"additions": 1708, "auther_ref": "clean-onlinedb-noexcept2", "auther_sha": "e0ee0e825f80bd687127c69954d9abce0e2441db", "author": "davidlt", "body": "ISO C++17 no more supports dynamic exception specifications. Starting\r\nC++11 noexcept and noexcept(expression) were introduced, where if\r\nexpression is false it means that function/method potentially can throw\r\nsomething. Simply put, now you can mark function as not throwing\r\nanything or potentially throwing something. Again, this was deprecated\r\nin C++11 and finally removed in C++17 to my understanding.\r\n\r\nSigned-off-by: David Abdurachmanov <David.Abdurachmanov@cern.ch>", "branch": "CMSSW_9_0_X", "changed_files": 367, "closed_at": "1485186655", "comments": 14, "commits": 1, "created_at": "1484835028", "deletions": 1812, "labels": ["comparison-available", "db-approved", "fully-signed", "orp-approved", "tests-approved"], "merge_commit_sha": "071e0d907e57184610d1d4852466eb2372ff39f9", "merged_at": "1485186655", "merged_by": "cmsbuild", "milestone": "CMSSW_9_0_X", "number": 17213, "release-notes": [], "review_comments": 0, "state": "closed", "title": "Replace throw specifier (deprecated) with noexcept (C++11)", "updated_at": "1485186655", "user": "davidlt"}